### PR TITLE
Include SetUUID for SSDP

### PR DESCRIPTION
**Pull request by laercionit**
Inclusion of the SetUUID Method for Custom UUID

### CODE REVIEW

File: libraries/ESP8266SSDP/ESP8266SSDP.cpp [hunk 1]
```diff
@@ -84,39 +81,23 @@ static const char _ssdp_schema_template[] PROGMEM =
...
+  "<deviceType>%s</deviceType>"
+  "<friendlyName>%s</friendlyName>"
+  "<presentationURL>%s</presentationURL>"
+  "<serialNumber>%s</serialNumber>"
+  "<modelName>%s</modelName>"
+  "<modelNumber>%s</modelNumber>"
+  "<modelURL>%s</modelURL>"
+  "<manufacturer>%s</manufacturer>"
+  "<manufacturerURL>%s</manufacturerURL>"
+  "<UDN>UUID: %s</UDN>"
```
[the remark this answers is not in the document]
Comment (laercionit) [reply libraries/ESP8266SSDP/ESP8266SSDP.cpp]: No specific purpose, just aesthetic.
It gets sharper for technical support information.

File: libraries/ESP8266SSDP/ESP8266SSDP.h [hunk 2]
```diff
@@ -39,7 +39,7 @@ class UdpContext;
 #define SSDP_SCHEMA_URL_SIZE        64
 #define SSDP_DEVICE_TYPE_SIZE       64
 #define SSDP_FRIENDLY_NAME_SIZE     64
-#define SSDP_SERIAL_NUMBER_SIZE     32
+#define SSDP_SERIAL_NUMBER_SIZE     37
```
[the remark this answers is not in the document]
Comment (laercionit) [reply libraries/ESP8266SSDP/ESP8266SSDP.h]: I increased to support larger serial numbers.
When I was testing, I felt this need.

Depending on the manufacturer, the serial number format may be somewhat similar to UUID in size.

File: libraries/ESP8266SSDP/ESP8266SSDP.cpp [hunk 3]
```diff
@@ -84,39 +81,23 @@ static const char _ssdp_schema_template[] PROGMEM =
...
-      "<friendlyName>%s</friendlyName>"
-      "<presentationURL>%s</presentationURL>"
-      "<serialNumber>%s</serialNumber>"
-      "<modelName>%s</modelName>"
-      "<modelNumber>%s</modelNumber>"
-      "<modelURL>%s</modelURL>"
-      "<manufacturer>%s</manufacturer>"
-      "<manufacturerURL>%s</manufacturerURL>"
-      "<UDN>uuid:%s</UDN>"
-    "</device>"
```
[the remark this answers is not in the document]
Comment (laercionit) [reply libraries/ESP8266SSDP/ESP8266SSDP.cpp]: I'm not 100% right either. As the code was commented, I found it appropriate to remove, but no problem to maintain. You decide.

[the remark this answers is not in the document]
Comment (devyte) [reply libraries/ESP8266SSDP/ESP8266SSDP.cpp]: Normally I'd agree, or I'd even request the removal myself, but I think this should be an exception. Please keep it for now, I consider this technical debt.

[the remark this answers is not in the document]
Comment (laercionit) [reply libraries/ESP8266SSDP/ESP8266SSDP.cpp]: Done!

**Pull request merged**